demo/app/App_Resources/Android/settings.json

```diff
@@ -0,0 +1 @@
+{"appId":"com.locusvisualarts.hiveku020719","minSdkVersion":null,"targetSdkVersion":null}
```

Comment:
We should avoid having specific app occurrences here. In this case, just delete the `settings.json`